src/test/java/calculator/CalculateTest.java

```diff
@@ -0,0 +1,62 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class CalculateTest {
+
+  private Calculator calculator;
+
+  @BeforeEach
+  void setup() {
+    calculator = new Calculator();
+  }
+
+  @ParameterizedTest
+  @CsvSource(value = {"2 + 3:5", "4 + 7:11"}, delimiter = ':')
+  @DisplayName("두수의 합을 구한다.")
```

Comment:
테스트의 설명이 불분명합니다. 
테스트를 설명할때는 이 테스트가 어떤 메서드를 어떤 상황에서 수행했을때 어떤 결과가 유추될지를 작성해줘야 합니다. 
```suggestion
  @DisplayName("유효한 두 값을 더할 경우, 두 값이 합해진 값이 반환된다.")
```